torch/testing/_internal/common_methods_invocations.py

```diff
@@ -281,6 +281,22 @@ def sample_inputs_addmm(op_info, device, dtype, requires_grad):
                                     low=None, high=None,
                                     requires_grad=False))),)
 
+def sample_inputs_broadcast_to(op_info, device, dtype, requires_grad):
+    test_cases = (
+        ((S, 1, 1), (S, S, S)),
+        ((S, 1, S), (S, S, S)),
+        ((S, 1), (S, S, S)),
+        ((1,), (S, S, S)),
+        ((1, S), (1, 1, S)),
+        ((), ()),
+        ((), (1, 3, 2)),
+    )
+
+    return [SampleInput((make_tensor(size, device, dtype,
```

Comment:
Return these as a tuple, not a list, because tuples in Python are immutable. Lists are appropriate only when a mutable iterable is needed.

Reply:
Thank you so much for pointing this out, I updated the `sample_inputs_broadcast_to` to return a `tuple`.